frontend/types/types.ts

```diff
@@ -0,0 +1,46 @@
+type MediaType = string;
+
+interface Album {
+  media: MediaType;
+  id: number;
+  title: string;
+  release_date: Date;
```

Comment:
question - how should we handle casing? i know TS convention is camelCase but i was having trouble implementing this since the API fields are snake_case which is our backend default

Reply:
I think snake_case is fine because our Supabase fields are in snake_case